packages-common/pluggable-widgets-tools/test-config/jest.config.js

```diff
@@ -14,6 +14,7 @@ const webConfig = {
     moduleFileExtensions: ["ts", "tsx", "js", "jsx"],
     setupFilesAfterEnv: [__dirname + "/test-index.js"],
     snapshotSerializers: ["enzyme-to-json/serializer"],
+    testMatch: ["<rootDir>/src/**/?(*.)(spec|test).[jt]s?(x)"],
```

Comment:
i think its a default

Reply:
No, it's not. This prevents e2e tests from being executed when `npm run test:unit` is executed in web widgets.